dapp/.env

```diff
@@ -19,11 +19,15 @@ VITE_SUBGRAPH_API_KEY=""
 # Get the API key from: 1Password.
 VITE_MEZO_PORTAL_API_KEY=""
 
+# Get the API key from: 1Password.
+VITE_ACRE_API_ENDPOINT=""
```

Comment:
I think we can set to `localhost:8788` by default. Ofc non-blcking comment.

Reply:
https://github.com/thesis/acre/pull/704/commits/11e86a198d93c57e3fa0605975d711c227de48a5